Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Melee/reverb_scythe.yml

```
@@ -0,0 +1,28 @@
+  name: resonant scythe
+  description: The answer seems to be here… But the fruit isnt sweet enough just yet.
+  parent: BaseItem
+  id: ReverbScythe
+  components:
+  - type: Sharp
+  - type: Sprite
+    sprite: _Goobstation/Objects/Weapons/Melee/breverb.rsi
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: 65
+    attackRate: 1.25
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)